### ISSUE TITLE
job 1-overlapper/blocks/000001.dat FAILED.

### ISSUE THREAD

**Issue by francoiskroll** [marbl/canu#1298]
I had some issues with my conda installation recently and had to reinstall packages etc. For some reason, I am having trouble running a `canu` command which used to run smoothly (see [issue 1282](https://github.com/marbl/canu/issues/1282)).

Command is:

```
canu \
  -p con -d canucon \
  -nanopore-raw pool.fastq \
  genomeSize=1.5k \
  minReadLength=500 minOverlapLength=250 \
  contigFilter="2 0 1.0 0.5 0" \
  corOutCoverage=100
```

I am getting:

> -- Detected Java(TM) Runtime Environment '10.0.1' (from 'java').
> dyld: Library not loaded: @rpath/./libjpeg.8.dylib
>   Referenced from: /Users/francoiskroll/miniconda3/bin/gnuplot
>   Reason: image not found
[...]
> ERROR:  Failed to run gnuplot from 'gnuplot'.ERROR:  Set option gnuplot=<path-to-gnuplot> or gnuplotTested=true to skip this test and not generate plots.

I tried installing/uninstalling/installing canu and/or gnuplot with conda a few times, I can't make it run.

So I tried adding the option `gnuplotTested=true` to my command.

Now it runs for a while, then throws an errror:

> -- 1 mhap precompute jobs failed:
> --   job 1-overlapper/blocks/000001.dat FAILED.
> [...]
> Canu release v1.5 failed with:
>   canu iteration count too high, stopping pipeline (most likely a problem in the grid-based computes)

This command was running with the same input files, so it has to be an installation/dependencies problem. Can you help?

**Comment by skoren** [comment]
Your gnuplot issue is because the install is missing a dynamic library, it wasn't properly installed. Your canu error is because you're on version 1.5 and have java 10 which doesn't support flags that canu 1.5 was using (see #909). Upgrade to a recent Canu release like 1.8 but DON'T use conda to do the upgrade. The conda packaging is broken so download the Canu 1.8 pre-compiled binaries for your system.

**Comment by francoiskroll** [comment]
Alright, solved both issues, thank you!

If anyone has similar issues (I'm on macOSX):

**gnuplot issue**
I had gnuplot installed by conda. I uninstalled it with `conda uninstall gnuplot` and installed with homebrew instead with `brew install gnuplot`.

**canu issue**
I uninstalled my conda version of canu (`conda uninstall canu`). I downloaded the latest release of canu in binary distribution (file `canu-1.8.Darwin-amd64.tar.xz`), extract it in whichever folder I put my packages in and added an entry in my /etc/paths (something like `.../canu-1.8/Darwin-amd64/bin`).
